backend/keystone.ts

```diff
@@ -22,15 +22,17 @@ export default withAuth(
       port: 8080,
     },
     db: {
-      provider: "mysql",
-      url: `mysql://${process.env.DB_USERNAME}:${process.env.DB_PASSWORD}@localhost:3306/${process.env.DB_NAME}`,
+      provider: "postgresql",
+      url: `postgresql://${process.env.DB_USERNAME}:${process.env.DB_PASSWORD}@localhost:5432/${process.env.DB_NAME}`,
```

Comment:
Can you undo these changes or make them use environment variables if you want to make this flexible for different engines; most of us are using MySQL and this will probably cause issues for anyone not using postgresql. We may need to all use the same engine, I'm not 100% sure what issues might arise if we don't.